Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -102,6 +102,9 @@ - (id)initWithAnalytics:(SEGAnalytics *)analytics
         dispatch_sync(dispatch_get_main_queue(), ^{
             self.flushTimer = [NSTimer scheduledTimerWithTimeInterval:30.0 target:self selector:@selector(flush) userInfo:nil repeats:YES];
         });
+        [self addSkipBackupAttributeToItemAtPath:self.userIDURL];
```

Comment:
oh good catch, I just copied the ones here https://github.com/segmentio/analytics-ios/blob/46a56f6fe0cd6866f3b6fac370d7815fc341aad0/Analytics/Classes/Internal/SEGSegmentIntegration.m#L498-L500 but look we're missing anonymousID there